File: Topic Based Problems/01 Fundamentals/Beginner/13 Square Root/C/SquareRoot.c
/**
 * @file SquareRoot.c
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Square Root in C
 * @version 1.0
 * @date 2023-09-24
 * 
 * Copyright (c) 2023
 * 
 */

#include <stdio.h>
#include <math.h>

int main() {
    double number;

    printf("Enter number: ");
    scanf("%lf", &number);

    printf("Square root of %.2lf is: %.2lf\n", number, sqrt(number));

    return 0;
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/13 Square Root/Java/SquareRoot.java
import java.util.Scanner;

public class SquareRoot {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double number;

        System.out.print("Enter number: ");
        number = scanner.nextDouble();

        System.out.printf("Square root of %.2f is: %.2f%n", number, Math.sqrt(number));

        scanner.close();
    }
}
